bridge/client/app/_components/ktb-evaluation-badge/ktb-evaluation-badge.component.scss

```diff
@@ -0,0 +1,59 @@
+@import 'client/variables';
+@import 'node_modules/@dynatrace/barista-components/core/src/style/variables';
+
+.badge {
+  padding: 2px 8px;
+  min-width: 30px;
+  height: 24px;
+  border-radius: 12px;
+  font-size: 12px;
+  color: black;
+  background-color: $gray-130;
+  text-align: center;
+
+  &.fill {
+    &.error {
+      background-color: $error-color;
+      color: $white;
+    }
+
+    &.warning {
+      background-color: $warning-color;
+      color: $white;
+    }
+
+    &.success {
+      background-color: $cta-color-active;
+      color: $white;
+    }
+  }
+
+  &.border {
+    background-color: $gray-200;
+    color: black;
+
+    &.error {
+      border: 1px solid $error-color;
+    }
+
+    &.warning {
+      border: 1px solid $warning-color;
+    }
+
+    &.success {
+      border: 1px solid $cta-color;
+    }
+  }
+}
+
+ktb-loading-spinner {
+  height: 20px;
+  width: 20px;
+  vertical-align: bottom;
+}
+
+/* fix incorrect height and width of the overlay */
+::ng-deep .cdk-overlay-connected-position-bounding-box {
```

Comment:
You might want to prefix this with `:host ::ng-deep ...`, or else this will be a global style applied also to other overlays on the same page.

Reply:
It has to be the global style because the overlay is not inside the child but on root level of the body

Reply:
As discussed offline, this can be left in.